elki/src/main/java/ch/ethz/globis/pht/PhDimFilter.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2011-2015 ETH Zurich. All Rights Reserved.
+ *
+ * This software is the proprietary information of ETH Zurich.
+ * Use is subject to license terms.
```

Comment:
Done, see e20927a94b98d03bc6b2e8bb73c891652782e9d6
I also stated that "This file is part of ELKI". Let me know in case this is not good.